src/Core/Models/SystemUser.cs

```diff
@@ -17,17 +17,15 @@ public class SystemUser
         /// GUID created by the "real" Authentication Component
         /// When the Frontend send a request for the creation of a new SystemUser the Id is null
         /// </summary>
-        [AllowNull]
-        [JsonPropertyName ("Id")]
-        public string Id { get; set; }
+        [JsonPropertyName("Id")]
+        public string Id { get; set; } = string.Empty;
 
         /// <summary>
         /// The Title and Description are strings set by the end-user in the Frontend.
         /// In the db this field is required, but if we use this model as a DTO, we allow null
         /// </summary>
-        [AllowNull]
         [JsonPropertyName("IntegrationTitle")]
```

Comment:
So, in what cases do you think this can be null or empty string? I just wonder as the db requires this data